ulog/README.md

```diff
@@ -1,8 +1,13 @@
 # Logging package
 
-ulog provides an API wrapper around the logging library (zap Logger)
-The logger is instantiated as logger with default options and can be configured
-via `Configure()` API and provided YAML configuration.
+`package ulog` provides an API wrapper around the logging library [zap](https://github.com/uber-go/zap).
+`ulog` uses the builder pattern to instantiate the logger. With `LogBuilder` you can perform pre-initialization setup
+by injecting configuration, custom logger, and log level prior to building the usable `ulog.Log` object. `ulog.Log`
+interface provides a few benefits:
+- Decouple services from the logger used underneath the framework.
```

Comment:
You don't need periods in this list. You only need them if a list item has two sentences in it (because then you need at least one period, and to stay consistent you add them everywhere).

Not a big deal, but it's a general rule to know.